docs/.github/workflows/nightly-release.yml

```
@@ -0,0 +1,66 @@
+name: Nightly Release
+
+on:
+  schedule:
+    # Runs at 00:00 UTC every day
+    - cron: '0 0 * * *'
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)